src/social-network-provider/twitter.com/utils/postBox.ts

```diff
@@ -1,35 +1,35 @@
 import { sleep, untilDocumentReady } from '../../../utils/utils'
-import { isUndefined } from 'lodash-es'
-import { newPostEditorFocusAnchor, newPostEditorHasFocus, newPostEditorSelector } from './selector'
+import { newPostEditorFocusAnchor, newPostEditorSelector } from './selector'
 import { equal } from '../../../utils/assert'
+import { untilElementAvailable } from '../../../utils/holoflowKits'
 
 export const postBoxInPopup = () => {
     return window.location.pathname.includes('compose')
 }
 
 export const getFocus = async () => {
     await untilDocumentReady()
-    while (isUndefined(newPostEditorSelector().evaluate())) {
-        await sleep(1000)
-    }
+    await untilElementAvailable(newPostEditorSelector())
```

Comment:
You should add a `timeout` when waiting a element to appear

Reply:
If there is no timeout, if the selector is not working in any situation, whole Maskbook will hang up because it is always waiting for the `untilElementAvailable`

Reply:
fixed inside untilElementAvailable